charts/aws-fluent-bit/values.yaml

```diff
@@ -1,18 +1,25 @@
 namespace:
-  create: true
+## Namespace creation is set to false as the namespace is expected to be created through the CLI since eksctl command may require.
+  create: false
   name: "calyptia-fluentbit"
 
+## The service account creation configuration is taken care by the eksctl command to contact the AWS marketplace API's to fetch the image.
 serviceAccount:
-  name: "eks-calyptia-fluentbit"
-  create: true
+  create: false
+## This is the name of the service account created from the eksctl create iamservice command.
+  name: irsa-calyptia-fluentbit
```

Comment:
why `irsa` prefix?

Reply:
the name provided here and the name used in the eksctl command should match.
I have changed both of these to sa-calyptia-fluentbit